packages/checkout/sdk/src/instance/instance.ts

```diff
@@ -56,14 +56,15 @@ export async function createExchangeInstance(
 ): Promise<Exchange> {
   const dexConfig = (await config.remote.getConfig(
     'dex',
-  )) as DexConfig;
+  )) as DexConfig | undefined;
 
   return new Exchange({
     chainId,
     baseConfig: new ImmutableConfiguration({
       environment: config.environment,
     }),
     overrides: dexConfig?.overrides,
+    secondaryFees: dexConfig?.secondaryFees,
```

Comment:
I was surprised that I needed to do this in the SDK + the widget, worth confirming that I've done this correctly 🙏